### ISSUE TITLE
mkremaster (Remaster me) task packages

### ISSUE THREAD

**Issue by unitylinux** [unity-linux/mklivecd#2]
When Unity-Linux was a distro we relied on task packages to pull in a specific group of packages ie. the unity-drivers package that pulled in a most common group of drivers needed for a livecd. Currently mkremaster with it's image from rpm feature still tries to call in these task packages though they (or equivalent versions) do not exist in Mageia. Not only are these task packages needed in porting these tools to Mageia, but they also help make the entire process (from rpm or running machine) easier as they offer an already grouped together selection for users who may not know exactly what is needed to get their image running on more than just the hardware of the machine of origin.

Long story short, task packages need to created in the Unity-Linux Repo and the repo should be pulled in by requires for mklivecd.
